### DISCUSSION TITLE
Remove underscores in folders?

### DISCUSSION THREAD

**Discussion by sojusnik** [peter88213/zim2obsidian#19] [General]
The current version removes underscores from notes, but not from folders. What is the rationale behind this?

Maybe it would be better to also remove underscores from folders, replacing them with spaces, as already done with notes? Thus, it would be easier to properly convert into Markdown style linking, since all spaces should be replaced there with `%20` as described [here](https://github.com/peter88213/zim2obsidian/issues/17).

**Comment by peter88213** [comment]
Strictly speaking, the script is not just removing underscores from file names, but renaming files according to the top first level heading (which is the Zim note name and can contain other characters that aren't allowed in file names). 

There is no further search for subdirectories with the note name yet. This is why the subdirectories aren't renamed. Be aware that renaming directories may require adjusting links to files that are not themselves renamed. This can be a non-trivial task.  
However, it could be done. I think, this topic is related to the "index page" topic (#6).
